### PR TITLE
Updated Board Changes

### PR DESCRIPTION
**Pull request by johnnavaj**
As discussed i have updated the board specification and the formatting. Please do the needful.

### CODE REVIEW

File: boards/artix7_35t.json [hunk 1]
```diff
@@ -1,36 +1,35 @@
...
+    "maximum_ram_size": 131072,
+    "maximum_size": 0,
     "protocol": "ftdi",
     "protocols": [
-      "ftdi",
-      "jlink"
-    ]
+		"ftdi",
+                "jlink"
+   ]
```
Comment (valeros) [inline boards/artix7_35t.json:32]: Bad formatting

File: examples/shakti-sdk_gpio-keypad/platformio.ini [hunk 2]
```diff
@@ -15,11 +15,20 @@ description =
 platform = shakti
 framework = shakti-sdk
 board = artix7_35t
-monitor_speed = 115200
+monitor_speed = 19200
+debug_tool = ftdi
+upload_protocol = custom
+upload_port = = /dev/ttyUSB*
+
+
```
Comment (valeros) [inline examples/shakti-sdk_gpio-keypad/platformio.ini:23]: Why do you need these fields?
```
debug_tool = ftdi
upload_protocol = custom
upload_port = = /dev/ttyUSB*
```
Also, the upload port shouldn't be specified here. What if a user has another device name? What about Windows users?

**Comment by valeros** [comment]
Seems like it's the same PR as before, could you please submit a clean PR with a single commit? Maybe it would be faster if I added these changes?

File: examples/shakti-sdk_gpio-keypad/platformio.ini (hunk 2 above)
Comment (johnnavaj) [reply examples/shakti-sdk_gpio-keypad/platformio.ini:23]: Currently everything is tested on linux only. So for now linux, later in a while will look into windows .

**Comment by johnnavaj** [comment]
No problem go ahead!

File: examples/shakti-sdk_gpio-keypad/platformio.ini (hunk 2 above)
Comment (johnnavaj) [reply examples/shakti-sdk_gpio-keypad/platformio.ini:23]: @valeros upload_protocol = custom is needed as it overrides the default ftdi configuration. Without this openocd won’t work .

Comment (valeros) [reply examples/shakti-sdk_gpio-keypad/platformio.ini:23]: What do you mean by "won't work"?  Can you open a new issue and describe the error you see when uploading?

**Pull request closed without merging**